db/migrate/20151029211323_create_users.rb

```diff
@@ -0,0 +1,14 @@
+class CreateUsers < ActiveRecord::Migration
+  def change
+    create_table :users do |t|
+      t.string :first_name, null: false
+      t.string :last_name, null: false
+      t.string :password_digest, null: false
```

Comment:
auth0 won't require us to have a password_digest; our users are going to log in through auth0, and we'll let them handle that.